docker-java/src/test/java/com/github/dockerjava/cmd/RestartContainerCmdImplIT.java

```diff
@@ -44,6 +49,43 @@ public void restartContainer() throws DockerException {
         dockerRule.getClient().killContainerCmd(container.getId()).exec();
     }
 
+    @Test
+    public void restartContainerWithSignal() throws Exception {
+        DefaultDockerClientConfig dockerClientConfig = DefaultDockerClientConfig.createDefaultConfigBuilder()
+            .withApiVersion(RemoteApiVersion.VERSION_1_42)
+            .withRegistryUrl("https://index.docker.io/v1/")
+            .build();
+        try (DockerClient dockerClient = createDockerClient(dockerClientConfig)) {
+            if (!getVersion(dockerClient).isGreaterOrEqual(RemoteApiVersion.VERSION_1_42)) {
```

Comment:
I think we can use [Assume](https://junit.org/junit4/javadoc/4.12/org/junit/Assume.html) instead.